AzureVMNoiseDataset2024.md

```diff
@@ -34,17 +40,25 @@ We provide the traces as they are, but are willing to help researchers understan
 ### Downloading
 
 You can download the dataset here:  [**vm-noise-data**](https://github.com/Azure/AzurePublicDataset/raw/master/data/vm-noise-data).
+
 ## Schema and Description
-### Description
- * 1 file per measurement unit: `test_suite=<test_suite>/test_name=<test_name>/vm_duration=<duration>/vm_region=<region>/vm_type=<type>/<unit>.csv`
+
+### Layout
+
+* 1 file per measurement unit, partitioned using Hive style table partitioning layout:
+
+`test_suite=<test_suite>/test_name=<test_name>/vm_lifespan=<short|long>/vm_region=<region>/vm_sku=<sku>/unit=<unit>.csv`
```

Comment:
This is a slightly different layout description than I think you used for the files, so we'd need to adjust the files as well.